### PR TITLE
Phase 1:  namex-ui-enhancements

### PR DESCRIPTION
**Pull request by scottrumsby**
entity tickets: #620-625, #627-628, #630

re-design of the name-examination user interface, per phase 1 requirements.  introduced vuetify, re-designed RequestInfoBanner, stdHeader, request editing/viewing and utilized new comments endpoint

*Issue #:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by katiemcgoff** [comment]
Issue: clicked "show" on Additional Info, the field is editable; should be view-only.
image

**Comment by katiemcgoff** [comment]
Issue: navigated from NR to search page, got vue error in console (no functional/obvious behaviour issue though):
image

File: client/src/components/application/Examine/nwpta/nwpta.vue [hunk 1]
```diff
@@ -189,6 +229,9 @@
       else this.requested_radio_selection = 'requested_nuans';
     },
     methods: {
+      logit() {
+        console.log('it happened')
```
Comment (katiemcgoff) [inline client/src/components/application/Examine/nwpta/nwpta.vue]: Please remove any console.log() statements that are clearly for debugging and not descriptive enough to help production debugging ('got here', '***', that kind of thing). Feel free to leave the ones that output, say, "got into ABC function" or an object that might be useful down the road for production debugging.

**Comment by katiemcgoff** [comment]
Issue: fonts don't seem to be right on the buttons (see screenshot):
image

**Comment by katiemcgoff** [comment]
Not sure about this one, @LJTrent please review - I can't find a link/screenshot of this specifically in the initial posting and can't find Scott Davis' mockups, if they even cover this level of detail.

Issue: 
Seems like the undo buttons/links on the names are too subtle, and or coloured incorrectly. Undo of an approved name seems okay, in that the link is blue:

image




**Undo link on rejected name is black, so really hard to see:** 


image



**The old ui for comparison:** 

image

**Comment by scottrumsby** [comment]
> Issue: clicked "show" on Additional Info, the field is editable; should be view-only.
> image
> 
> @scottrumsby - This seems to happen when you select edit request and then come back to the banner and select +Show on Additional Info. It appears in Edit mode. Can you please correct this.


> Issue: navigated from NR to search page, got vue error in console (~no functional/obvious behaviour issue though~) with no results:
> image
> 
> Follow-up: this may not have to do with navigating back to search page, but rather having the search input empty. I won't spend time debugging, will leave that to developer.
> 
> @katiemcgoff Only seems to be when you select All in the search filter. I see the Vue error when I look in F12 and it does eventually return all rows. Takes about 4 minutes. @scottrumsby. Please check this.
> all-search

I have solved this issue but FYI it was an existing issue (ie, master exhibits the same behaviour)

File: client/src/components/application/Examine/nwpta/nwpta.vue (hunk 1 above)
Comment (scottrumsby) [reply client/src/components/application/Examine/nwpta/nwpta.vue]: oops, yes, I meant to delete that one.

**Comment by scottrumsby** [comment]
> Issue: fonts don't seem to be right on the buttons (see screenshot):
> image
> 
> @katiemcgoff I think the Fonts are fine on the buttons. Scott got the CSS form the last mock-up site. I

I totally forgot that .svg files will leverage the user agents installed fonts to render text.  I should address this issue as it will cause inconsistency.  Users who have the particular font installed on their computer locally will see the buttons rendered with it and others will see a system fall back, which appears to be a serif font in some cases.  I will just rasterize the text-containing buttons to .png format and this won't be an issue at all.

**Comment by LJTrent** [comment]
> > Issue: fonts don't seem to be right on the buttons (see screenshot):
> > image
> > @katiemcgoff I think the Fonts are fine on the buttons. Scott got the CSS form the last mock-up site. I
> 
> I totally forgot that .svg files will leverage the user agents installed fonts to render text. I should address this issue as it will cause inconsistency. Users who have the particular font installed on their computer locally will see the buttons rendered with it and others will see a system fall back, which appears to be a serif font in some cases. I will just rasterize the text-containing buttons to .png format and this won't be an issue at all.

Ok.  Great.  Glad you looked at this.

**Comment by katiemcgoff** [comment]
Edit Details - 2 issues:
1. The new Additional Info save and cancel buttons are there (yay) but shouldn't be in the bigger edit mode.
2. The save and cancel buttons for the bigger edit are missing (probably because of replacing the svgs with pngs).

image

**Comment by katiemcgoff** [comment]
Comments and Additional Info (and maybe NOB) need to show line breaks. See the comments list in decision screen for an example.

image

**Comment by katiemcgoff** [comment]
New issue: clicking cancel shows the cancel popup (existing from previous UI) but doesn't manage the modal screen disabling correctly.

image

**Comment by katiemcgoff** [comment]
> New issue: clicking cancel shows the cancel popup (existing from previous UI) but doesn't manage the modal screen disabling correctly.
> 

Same problem for customer message popup:
image

**Comment by katiemcgoff** [comment]
Okay all good. I'm approving and merging.

**Pull request merged**